File: 0744-find-smallest-letter-greater-than-target/0744-find-smallest-letter-greater-than-target.java
class Solution {
    public char nextGreatestLetter(char[] letters, char target) {
        
//         for(int i =0; i<letters.length; i++){
//             if(char[i] - '0' >target){
//                 return char[i];
//             }
//         }
            
        for(char ch:letters){
            if(ch >target){
                return ch;
            }
        }
        return letters[0];
    }
}

File: 0744-find-smallest-letter-greater-than-target/0744-find-smallest-letter-greater-than-target.js
/**
 * @param {character[]} letters
 * @param {character} target
 * @return {character}
 */
var nextGreatestLetter = function (letters, target) {
  return letters.find((item) => item > target)
    ? letters.find((item) => item > target)
    : letters[0];
};
